core/app/models/spree/promotion/rules/item_total.rb

```diff
@@ -6,20 +6,36 @@ module Rules
       # A rule to apply to an order greater than (or greater than or equal to)
       # a specific amount
       class ItemTotal < PromotionRule
+        include ActiveSupport::Deprecation::DeprecatedConstantAccessor
+
         preference :amount, :decimal, default: 100.00
         preference :currency, :string, default: ->{ Spree::Config[:currency] }
         preference :operator, :string, default: '>'
 
-        OPERATORS = ['gt', 'gte']
+        OPERATORS_MAP = {
```

Comment:
Is there any reason to don't use a method instead of a constant if we are deprecating this?

Reply:
No reason, I used a constant just because there was a constant before, I'll change it to a method 👍